src/main/java/io/cdap/plugin/salesforce/authenticator/AuthenticatorCredentials.java

```diff
@@ -47,12 +47,12 @@ public AuthenticatorCredentials(String username, String password,
   }
 
   private AuthenticatorCredentials(@Nullable OAuthInfo oAuthInfo,
-                                   @Nullable String username,
-                                   @Nullable String password,
-                                   @Nullable String consumerKey,
-                                   @Nullable String consumerSecret,
-                                   @Nullable String loginUrl,
-                                   @Nullable Integer connectTimeout) {
+                                  @Nullable String username,
```

Comment:
Looks like an unintended change. can you revert this ?

Reply:
done